src/index.html

```diff
@@ -9,6 +9,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="box">
+        <div class="image">
+          <link rel="stylesheet" href="./images/logo.png">
```

Comment:
it should be an `a `element with `img `inside. `link `we usually use in the `head`